### PR TITLE
[ACS] `az aks create/update`: Ephemeral disk additional support for Azure Container Storage

### PR DESCRIPTION
**Pull request by mukhoakash**
**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/azure-cli/azure/cli/command_modules/acs/azurecontainerstorage/_helpers.py [hunk 1]
```diff
@@ -144,7 +149,7 @@ def get_extension_installed_and_cluster_configs(
     resource_group,
     cluster_name,
     agentpool_profiles
-) -> Tuple[bool, bool, bool, bool, float]:
+) -> Tuple[bool, bool, bool, bool, float, str, str]:
```
Comment (FumingZhang) [inline src/azure-cli/azure/cli/command_modules/acs/azurecontainerstorage/_helpers.py:152]: actually returned 5 bools?
> ) -> Tuple[bool, bool, bool, bool, bool, float, str, str]:

File: src/azure-cli/azure/cli/command_modules/acs/azurecontainerstorage/_helpers.py [hunk 2]
```diff
@@ -276,6 +303,7 @@ def get_current_resource_value_args(
 def get_desired_resource_value_args(
```
Comment (FumingZhang) [inline src/azure-cli/azure/cli/command_modules/acs/azurecontainerstorage/_helpers.py:303]: This is different from the impletion in aks-preview, missing `perf_tier_updated`, expected?

Comment (mukhoakash) [reply src/azure-cli/azure/cli/command_modules/acs/azurecontainerstorage/_helpers.py:307]: Resolved

File: src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py [hunk 3]
```diff
@@ -6445,10 +6473,6 @@ def set_up_azure_container_storage(self, mc: ManagedCluster) -> ManagedCluster:
             )
```
[the remark this answers is not in the document]
Comment (mukhoakash) [reply src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py:6473]: Resolved.

File: src/azure-cli/azure/cli/command_modules/acs/azurecontainerstorage/_helpers.py (hunk 1 above)
Comment (mukhoakash) [reply src/azure-cli/azure/cli/command_modules/acs/azurecontainerstorage/_helpers.py]: Resolved.

**Comment by mukhoakash** [comment]
@FumingZhang Thanks for pointing out the mismatch with aks-preview. Fixed the neccessary ones and updated the PR.

**Review by FumingZhang — COMMENTED** [review]
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=96750971&view=results)
- test_aks_create_with_azurecontainerstorage
- test_aks_update_with_azurecontainerstorage

**Comment by mukhoakash** [comment]
Requeued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=96778017&view=results):
- test_aks_create_with_azurecontainerstorage
- test_aks_update_with_azurecontainerstorage
- test_aks_create_with_azurecontainerstorage_with_nodepool_name
- test_aks_create_with_azurecontainerstorage_with_ephemeral_disk_parameters

**Review by FumingZhang — APPROVED** [review]
LGTM

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**